src/ui/NoNameFacet/NoNameFacetHeader.ts

```diff
@@ -88,23 +92,66 @@ export class NoNameFacetHeader {
   }
 
   private recreateOperator() {
-    $$(this.operatorElement).replaceWith(this.createOperator());
+    const newElement = this.createOperator();
+    $$(this.operatorElement).replaceWith(newElement);
+    this.operatorElement = newElement;
+  }
+
+  private createCollapse() {
+    const { enableTogglingCollapse, isCollapsed } = this.options.rootFacetOptions;
+    const svgIcon = isCollapsed ? SVGIcons.icons.facetExpand : SVGIcons.icons.facetCollapse;
+    const collapseBtn = $$('button', { className: 'coveo-facet-header-collapse' }, svgIcon);
+    const className = `coveo-facet-settings-section-${isCollapsed ? 'show' : 'hide'}-svg`;
+    SVGDom.addClassToSVGInContainer(collapseBtn.el, className);
+
+    const label = isCollapsed ? l('Expand') : l('Collapse');
+    collapseBtn.setAttribute('aria-label', label);
+    collapseBtn.setAttribute('title', label);
```

Comment:
Is it useful to include a title attribute whose value is equal to the aria-label? In my experience, this does not sound nice on a screen-reader which says: e.g.
"Expand button Expand"

Unless you want to provide additional info in the title attribute, I think the aria-label is enough.

Reply:
The title is more for when hovering the button, you see what it does. The other buttons have those title.
With voiceover I don't hear "Expand button Expand" just "Expand button". Maybe on your software it sound different